spring-rabbit/src/main/java/org/springframework/amqp/rabbit/support/micrometer/RabbitMessageSenderContext.java

```diff
@@ -33,11 +33,17 @@ public class RabbitMessageSenderContext extends SenderContext<Message> {
 
 	private final String destination;
 
-	public RabbitMessageSenderContext(Message message, String beanName, String destination) {
+	private final String exchange;
+
+	private final String routingKey;
+
+	public RabbitMessageSenderContext(Message message, String beanName, String exchange, String routingKey) {
```

Comment:
This is a breaking change.
Please, consider to deprecate an existing ctor and introduce a new one.
The deprecated one has to call a new one via splitting on that `/`.
Just in case someone uses the ctor we just deprecating.

Reply:
I set `null` value if someone use the old ctor

I think we should not split `/` in the old ctor.

For example
`destination = exchange + "/" + routingKey`
exchange = `//`
routingKey = `//`
the destination = `/////`

RabbitMQ does not validate the '/` character when we give a name to the queue or the exchange, and it's so hard to determine our `/`

Reply:
Yes. That makes sense.
Thank you!